app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/ClientHomeViewModel.kt

```diff
@@ -179,4 +181,17 @@ class ClientHomeViewModel @Inject constructor(
             .subscribeBy(onError = { Timber.e(it) })
             .addTo(compositeDisposable)
     }
+
+    override fun receivedMessages(): Observable<Message> {
+        return rxWebSocketClient.events()
+            ?.ofType(RxWebSocketClient.Event.Message::class.java)
+            ?.flatMap { webSocketMessage ->
+                val message = messageParser.parseWebSocketMessage(webSocketMessage)
+                message?.let {
+                    Observable.just(it)
+                } ?: Observable.error(Throwable("Failed Initialisation"))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/89ff9ffb603a83b79e177dd00519720c6a8d52ac/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/ClientHomeViewModel.kt#L192"></span>Multiple occurrences of the same string literal within a single file detected.</td>
    </tr>
  </tbody>
</table>